financial-connections/src/main/java/com/stripe/android/financialconnections/image/StripeImage.kt

```diff
@@ -0,0 +1,84 @@
+package com.stripe.android.financialconnections.image
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.BoxWithConstraintsScope
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ImageBitmap
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.graphics.painter.BitmapPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.Dp.Companion.Infinity
+import androidx.compose.ui.unit.IntSize.Companion.Zero
+import com.stripe.android.financialconnections.ui.LocalImageLoader
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+
+@Composable
+internal fun StripeImage(
```

Comment:
Could we move this to `stripe-core` so that they can be reused?

Reply:
Agreed, I think we will need this for UPE

Reply:
I don’t quite see it fitting in with `stripe-core`. We already have `payments-ui-core` — could that work? Or maybe it warrants a `stripe-ui-core` module?

Reply:
> Or maybe it warrants a stripe-ui-core module?

I'm more leaning towards this. We don't want non-UI modules to depend on image loading components, or transitive dependencies (compose, disk cache, etc).

This would also open the door to shareable UI components.

I can tackle this on a separate PR if we like the idea (@ccen-stripe  @jameswoo-stripe)

Reply:
moved this to `stripe-ui-core`.